unicode_smilies/unicode_smilies.php

```diff
@@ -1,260 +1,2042 @@
+//Smilies::add($b, ':hammock', '&#x;');
+//Smilies::add($b, ':pillow', '&#x;');
+//Smilies::add($b, ':yawn', '&#x;');
 
 // Fight/Flame/Violent smileys
 
-//	Smilies::add($b, ':2guns', '&#x;');
-//	Smilies::add($b, ':alienfight', '&#x;');
-//	Smilies::add($b, ':army', '&#x;');
-//	Smilies::add($b, ':arrowhead', '&#x;');
-//	Smilies::add($b, ':bfg', '&#x;');
-//	Smilies::add($b, ':bowman', '&#x;');
-//	Smilies::add($b, ':chainsaw', '&#x;');
-//	Smilies::add($b, ':crossbow', '&#x;');
-//	Smilies::add($b, ':crusader', '&#x;');
-//	Smilies::add($b, ':dead', '&#x;');
-//	Smilies::add($b, ':hammersplat', '&#x;');
-//	Smilies::add($b, ':lasergun', '&#x;');
-//	Smilies::add($b, ':machinegun', '&#x;');
-//	Smilies::add($b, ':acid', '&#x;');
+//Smilies::add($b, ':2guns', '&#x;');
+//Smilies::add($b, ':alienfight', '&#x;');
+//Smilies::add($b, ':army', '&#x;');
+//Smilies::add($b, ':arrowhead', '&#x;');
+//Smilies::add($b, ':bfg', '&#x;');
+//Smilies::add($b, ':bowman', '&#x;');
+//Smilies::add($b, ':chainsaw', '&#x;');
+//Smilies::add($b, ':crossbow', '&#x;');
+//Smilies::add($b, ':crusader', '&#x;');
+//Smilies::add($b, ':dead', '&#x;');
+//Smilies::add($b, ':hammersplat', '&#x;');
+//Smilies::add($b, ':lasergun', '&#x;');
+//Smilies::add($b, ':machinegun', '&#x;');
+//Smilies::add($b, ':acid', '&#x;');
 
 // Fantasy smileys - monsters and dragons fantasy.  The other type of fantasy belongs in adult smileys
 
-	Smilies::add($b, ':alienmonster', '&#x1F47E;');
-//	Smilies::add($b, ':barbarian', '&#x;');
-//	Smilies::add($b, ':dinosaur', '&#x;');
-	Smilies::add($b, ':dragon', '&#x1F409;');
-	Smilies::add($b, ':draco', '&#x1F409;');
-	Smilies::add($b, ':ghost', '&#x1F47B;');
-//	Smilies::add($b, ':mummy', '&#x;');
+Smilies::add($b, ':alienmonster', '&#x1F47E;');
+//Smilies::add($b, ':barbarian', '&#x;');
+//Smilies::add($b, ':dinosaur', '&#x;');
+Smilies::add($b, ':dragon', '&#x1F409;');
+Smilies::add($b, ':draco', '&#x1F409;');
+Smilies::add($b, ':ghost', '&#x1F47B;');
+//Smilies::add($b, ':mummy', '&#x;');
 
 // Food smileys
 
-	Smilies::add($b, ':apple', '&#x1F34E;');
-//	Smilies::add($b, ':broccoli', '&#x;');
-	Smilies::add($b, ':cake', '&#x1F370;');
-//	Smilies::add($b, ':carrot', '&#x1F955;'); // Bad client support
-	Smilies::add($b, ':popcorn', '&#x1F37F;');
-	Smilies::add($b, ':tomato', '&#x1F345;');
-	Smilies::add($b, ':banana', '&#x1F34C;');
-	Smilies::add($b, ':cooking', '&#x1F373;');
-	Smilies::add($b, ':fryegg', '&#x1F373;');
-	Smilies::add($b, ':birthdaycake', '&#x1F382;');
+Smilies::add($b, ':apple', '&#x1F34E;');
+//Smilies::add($b, ':broccoli', '&#x;');
+Smilies::add($b, ':cake', '&#x1F370;');
+//Smilies::add($b, ':carrot', '&#x1F955;'); // Bad client support
+Smilies::add($b, ':popcorn', '&#x1F37F;');
+Smilies::add($b, ':tomato', '&#x1F345;');
+Smilies::add($b, ':banana', '&#x1F34C;');
+Smilies::add($b, ':cooking', '&#x1F373;');
+Smilies::add($b, ':fryegg', '&#x1F373;');
+Smilies::add($b, ':birthday cake', '&#x1F382;');
```

Comment:
Do spaces within the smiley definition work?

Reply:
It works on my instance